manifests/c/CloudImperiumGames/RSILauncher/1.5.2/CloudImperiumGames.RSILauncher.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CloudImperiumGames.RSILauncher
+PackageVersion: 1.5.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64

```

Comment:
Azure Pipelines claims that silent install doesn't work, but RSI-Setup-1.5.2.exe /S installs silently here.

Star Citizen is strictly x64 only, so maybe the installer is failing on a sanity check, and Azure Pipelines thinks it's a silent installation failure?

Reply:
This'll teach me to be lazy and use Wingetcreate.